migration/registry/from-mi-1.0.0-to-later/README.md

```diff
@@ -0,0 +1,37 @@
+#Registry Migration Tool for WSO2 Micro Integrator
```

Comment:
Hi @tharinduwijewardane 

Seems the Mark Downs are not applied correctly ( https://github.com/wso2/micro-integrator/tree/master/migration/registry/from-mi-1.0.0-to-later ) . Shall we fix the readme file to look better.
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers

Reply:
+1. We need to add a space after `#` in headings.

Reply:
Fixed in https://github.com/wso2/micro-integrator/pull/246. Please approve.